uk_bin_collection/uk_bin_collection/councils/ValeofGlamorganCouncil.py

```
@@ -98,16 +70,14 @@ def parse_data(self, page: str, **kwargs) -> dict:
                             bin_date,
                         )
                     )
-                except Exception as ex:
+                except Exception:
                     continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)